### PR TITLE
basic debian layout

### PR DESCRIPTION
**Pull request by ctr49**
(no postinst and rules yet)
#353

### CODE REVIEW

**Comment by ctr49** [comment]
another update accidentally slipped into this pull request, reverted now

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Package: cometvisu
+Replaces: cometvisu-wiregate,cometvisu-0.5,cometvisu-0.8
+Priority: extra
+Maintainer: CometVisu.org <info@cometvisu.org>
+Vcs-Git: https://github.com/CometVisu/CometVisu.git
+Vcs-browser: https://github.com/CometVisu/CometVisu
+Homepage: http://www.cometvisu.org/wiki/CometVisu
```
Comment (ChristianMayer) [inline debian/control]: Should the homepage not be the more generic and correct  http://www.cometvisu.org/

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Architecture: all
+Section: net
+Depends: libapache2-mod-php5 | libapache-mod-php5 | php5-cgi | php5, rrdtool (>= 1.3.1-4+nmu1), cometvisu-backend | knxd-tools
```
Comment (ChristianMayer) [inline debian/control]: php is only recomended, not required. (Only plugins can depend on it, all basic features should work on a simple system w/o PHP installed, like a little router)

File: debian/copyright [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cometvisu
+Source: https://github.com/CometVisu/CometVisu
+
+Files: *
+Copyright: Christian Mayer <CometVisu@ChristianMayer.de>
+           Tobias Bräutigam <tbraeutigam@gmail.com>
+           Michael Markstaller <devel@wiregate.de> / Elaborated Networks GmbH
```
Comment (ChristianMayer) [inline debian/copyright]: Makki is not associated with Elaborated Networks anymore.

Comment (Makki1) [inline debian/copyright]: On 03.07.2016 21:08, Christian Mayer wrote:
> +Copyright: Christian Mayer <CometVisu@ChristianMayer.de>
>> +           Tobias Bräutigam <tbraeutigam@gmail.com>
>> +           Michael Markstaller <devel@wiregate.de> / Elaborated Networks GmbH
> 
> Makki is not associated with Elaborated Networks anymore.


Thats correct though I don't really care in terms of formulating the (C)
- I wont change it just for this..
If you do changes, feel free to use

Michael Markstaller <michael@markstaller.de>


-- 
Für Rückfragen stehe ich jederzeit gerne zur Verfügung.

Mit freundlichen Grüssen

Michael Markstaller
Softwareentwicklung & IT-Dienstleistungen

Am Birkengarten 19b
82024 Taufkirchen
Tel: +49-89-21553825, Mobile: +49-179-2080789
USt-ID: DE293191950

File: debian/control (hunk 1 above)
Comment (ctr49) [inline debian/control]: changed

File: debian/control (hunk 2 above)
Comment (ctr49) [inline debian/control]: I took this part from the Wiregate deb package control file, but in general I'd rather make php a dependency, at least on a Debian system (where this control file is being used) due to config-check and editor.

File: debian/copyright (hunk 3 above)
Comment (ctr49) [inline debian/copyright]: I just went through some file headers and copied the copyright notices from there. The actual syntax should have years, and then it makes perfect sense again to leave it as is (for a period 201x - 2014?) and add the new address for the period afterwards.

**Comment by ChristianMayer** [comment]
Should the file src/plugins/rsslog/rsslog_external.php really be in this pull request?

**Comment by ctr49** [comment]
No, I just forgot to create an own branch before starting the pull request, so subsequent changed made it into this one. But the change is already merged, so it shouldn't really hurt.

**Pull request merged**